inc/admin/admin.php

```diff
@@ -487,6 +489,75 @@ function rocket_analytics_optin() {
 }
 add_action( 'admin_post_rocket_analytics_optin', 'rocket_analytics_optin' );
 
+/**
```

Comment:
Could you create a new subscriber class and add these 2 methods in it instead, that way we continue to shift to OOP code